transform/transformers/PCKTransformer.py

```diff
@@ -4,13 +4,15 @@
 
 class PCKTransformer(object):
     form_ids = {
```

Comment:
We should change _form_ids_ here to _form_types_. That means:

- instrument_id <--> form_id (numerical strings)
- (survey_id, form_id) -> form_type (string codes for forms which appear in PCK file.)